### ISSUE TITLE
GitHub Actions fails to push container image to GHCR (`permission_denied: write_package`)

### ISSUE THREAD

**Issue by eecavanna** [microbiomedata/nmdc-edge#165]
#### Summary

GitHub Actions fails to push container image to GHCR (`permission_denied: write_package`)

#### Log

- https://github.com/microbiomedata/nmdc-edge/actions/runs/9134747885

#### Screenshot

<img width="662" alt="image" src="https://github.com/microbiomedata/nmdc-edge/assets/134325062/7c0b7351-c8f5-4b40-81b6-1d416e4d80c0">

**Comment by eecavanna** [comment]
Here's a Stack Overflow conversation about this topic:

https://stackoverflow.com/a/72585915

**Comment by eecavanna** [comment]
This is working now.

Here's what I did to fix it:
- Go to the "Package Settings" page at: https://github.com/orgs/microbiomedata/packages/container/nmdc-edge-web-app/settings.
- Scroll down to "Manage Actions access" and click the "Add Repository" button (to "pick the repositories that can access this package using GitHub Actions").
- Select the `nmdc-edge` repository in the list and click the "Add repositories" button.
- Once the "Package Settings" page reloads and shows `nmdc-edge` in the "Manage Actions access" section, click the "Role" menu on the same row and change the role from `Read` to `Write`.

Once I did that, I re-ran the GitHub Actions workflow and saw it successfully pushed an image to GitHub Container Registry. 🙌 

![Image](https://github.com/microbiomedata/nmdc-edge/assets/134325062/a72a2a97-8b38-46cb-8385-9044633958ba)

Note: I also deleted the `v0.1.0-alpha.1` Release and tag (from last week's failed build) and then re-used that same name for today's Release and tag. Here's a link to the Release: https://github.com/microbiomedata/nmdc-edge/releases/tag/v0.1.0-alpha.1
